interbotix_ros_xscobots/interbotix_xscobot_moveit/launch/moveit_api.launch.py

```diff
@@ -0,0 +1,355 @@
+                },
+                'use_sim_time': use_sim_time_param,
+            },
+            moveit_config.robot_description,
+            moveit_config.robot_description_semantic,
+            moveit_config.robot_description_kinematics,
+            moveit_config.planning_pipelines,
+            moveit_config.joint_limits,
+            moveit_config.trajectory_execution,
+        ],
+        remappings=remappings,
+        output={'both': 'screen'},
+    )
+
+    rviz_node = Node(
+        package='rviz2',
+        executable='rviz2',
+        name='rviz2',
+        output={'both': 'log'},
+        arguments=[
+            '-d',
+            rviz_config_file,
+            '-f',
+            rviz_frame_launch_arg,
+        ],
+        parameters=[
+            moveit_config.robot_description,
+            moveit_config.robot_description_semantic,
+            moveit_config.robot_description_kinematics,
+            moveit_config.planning_pipelines,
+            moveit_config.joint_limits,
+        ],
+        remappings=remappings,
+    )
+
+    static_tf = Node(
+        package='tf2_ros',
+        executable='static_transform_publisher',
+        name='static_transform_publisher',
+        output='log',
+        arguments=[
+            '0.0',
+            '0.0',
+            '0.0',
+            '0.0',
+            '0.0',
+            '0.0',
+            'world',
+            'dx400/base_link',
+        ],
+    )
+
+    xscobot_ros_control_launch_include = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare('interbotix_xscobot_ros_control'),
+                        'launch',
+                        'xscobot_ros_control.launch.py',
+                    ]
+                )
+            ]
+        ),
+        launch_arguments={
+            'robot_model': robot_model_launch_arg,
+            'robot_name': robot_name_launch_arg,
+            'base_link_frame': base_link_frame_launch_arg,
+            'show_gripper_fingers': 'true',
+            'show_ar_tag': show_ar_tag_launch_arg,
+            'use_world_frame': use_world_frame_launch_arg,
+            'external_urdf_loc': external_urdf_loc_launch_arg,
+            'use_rviz': 'false',
+            'mode_configs': mode_configs_launch_arg,
+            'hardware_type': hardware_type_launch_arg,
+            'robot_description': robot_description_launch_arg,
+            'use_sim_time': use_sim_time_param,
+            'xs_driver_logging_level': xs_driver_logging_level_launch_arg,
+        }.items(),
+        condition=IfCondition(
+            PythonExpression(
+                ["'", hardware_type_launch_arg, "'", ' in ("actual", "fake")']
+            )
+        ),
+    )
+    return [
+        TimerAction(
+                period=5.0,
+                actions=[moveit_py_node],
+                    ),
```

Comment:
Formatting is still weird here.
```suggestion
        TimerAction(
            period=5.0,
            actions=[moveit_py_node],
        ),
```